crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -24,6 +24,17 @@ pub enum AppRequest {
     /// [`AppResponse::AppInfo`]
     AppInfo,
 
+    /// Request information about the agents in this Conductor's peer store.
+    /// This is limited to cells of the app you are connected to.
```

Comment:
The first line is rendered as a summary on rust docs. If you write two sentences like this then they run together in the summary and don't display well. Docs should be a short summary, followed by a block of any details and the details can render as a paragraph

```suggestion
    /// Request information about the agents in this Conductor's peer store.
    ///
    /// This is limited to cells of the app you are connected to.
```